demos/js/console.js

```diff
@@ -0,0 +1,258 @@
+/*
+ * Copyright 2022, Tony Atkins
+ * Copyright 2011-2020, Colin Clark
+ *
+ *  Licensed under the MIT license, see LICENSE for details.
+ */
+(function (fluid) {
+    "use strict";
+    var youme = fluid.registerNamespace("youme");
+
+    fluid.defaults("youme.demos.console.textareaWithError", {
+        gradeNames: ["youme.templateRenderer"],
+        markup: {
+            container: "<div class='console-textarea-with-error'><textarea class='editor'>%editorString</textarea><div class='error'>%errorString</div></div>"
+        },
+
+        model: {
+            errorString: false,
+            editorString: ""
+        },
+
+        selectors: {
+            error: ".error",
+            editor: ".editor"
+        },
+
+        modelListeners: {
+            errorString: [
+                {
+                    excludeSource: "init",
+                    this: "{that}.dom.error",
+                    method: "text",
+                    args: ["{that}.model.errorString"]
+                },
+                {
+                    this: "{that}.dom.error",
+                    method: "toggleClass",
+                    args: ["console-visible", "{change}.value"]
+                }
+            ],
+            editorString: {
+                excludeSource: "init",
```

Comment:
Also, this is the only one that didn't do what I expected when converting to a `modelRelay`.  The initial value is empty.

Reply:
I pushed up what I have, you can see that the component (renamed to `midiConsole` to avoid the previous goof of clobbering the global `console`) has the editor strings in its model, and that the individual components each have their editor string, but nothing is displayed.

Reply:
Just checking on the status of this - https://issues.fluidproject.org/browse/FLUID-6746 got accidentally closed due to a typo in the commits for https://github.com/fluid-project/infusion-docs/commit/b37297e84c8964e81929c7f2317ed8f49b8a398c to infusion-docs which swapped FLUID-6476 for FLUID-6746 - FLUID-6746 should be listed as still outstanding as highlighted by this relay problem